src/blocks/EmbedBlock/embeds/Default.astro

```diff
@@ -31,7 +31,7 @@ const maxWidth = data.width ? `${data.width}px` : '100%';
   </slot>
   <slot name="noscript">
     { scripts.map(({ src, ...attributes }) => (
-      <script data-src={ src } {...attributes} />
+      <script is:inline data-src={ src } {...attributes} />
```

Comment:
I imagine the added `is:inline` in multiple files is due to the `@astro/check` upgrade. Doesn't change the implementation, so all good.